spiel/st/m0t1fs_spiel_dix_common_inc.sh

```diff
@@ -136,8 +136,8 @@ EOF
 
 spiel_dix_repair_abort()
 {
-echo $M0_SRC_DIR/utils/spiel/m0spiel $SPIEL_OPTS
-    $M0_SRC_DIR/utils/spiel/m0spiel $SPIEL_OPTS <<EOF
+echo "$M0_SRC_DIR"/utils/spiel/m0spiel "$SPIEL_OPTS"
```

Comment:
PLEASE NOTE HERE.

This change might be wrong.
`echo "$M0_SRC_DIR"/utils/spiel/m0spiel $SPIEL_OPTS`
The `$SPIEL_OPTS` should not be double quoted.

I have clarified this many times.

Reply:
done